BootloaderCommonPkg/Include/Library/CryptoLib.h

```diff
@@ -189,6 +187,30 @@ RsaVerify_Pkcs_1_5 (
   CONST UINT8              *Hash
   );
 
+
+/**
+  Verifies the RSA signature with PSS encoding scheme defined in RSA PSS.
+
+  @param[in]  PubKeyHdr         Pointer to a PubKey data.
+  @param[in]  SignatureHdr      Pointer to signature data to be verified.
+  @param[in]  Src               Pointer to meassage.
+  @param[in]  SrcSize           Size of the message.
+
+  @retval  RETURN_SUCCESS             Valid signature.
+  @retval  RETURN_INVALID_PARAMETER   Key or signature format is incorrect.
+  @retval  RETURN_SECURITY_VIOLATION  Invalid signature.
+
+**/
+
+RETURN_STATUS
+RsaVerify_PSS (

```

Comment:
Should we consider using generic name for function ?    EX:  SignatureVerify (...)
Other info should be indicated in the parameter, PUB_KEY_HDAR and SIGNATURE_HDR.